pkg/types/types.go

```diff
@@ -57,3 +57,17 @@ type MedianProvider interface {
 	ReportCodec() median.ReportCodec
 	MedianContract() median.MedianContract
 }
+
+type OCR2VRFRelayer interface {
+	Relayer
+	NewDKGProvider(rargs RelayArgs, transmitterID string) (DKGProvider, error)
+	NewOCR2VRFProvider(rargs RelayArgs, transmitterID string) (OCR2VRFProvider, error)
+}
+
+type DKGProvider interface {
+	Plugin
+}
+
+type OCR2VRFProvider interface {
```

Comment:
Lets provide some boilerplate godoc comments to these. They start with the name of the type and some description. For example:

```go
// DKGProvider provides all components needed for a DKG OCR2 plugin
type DKGProvider interface {
// ...
```